components/Blog/BlogListNavigationButtons.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+import Button from '../Button'
+import Link from '../../components/Link'
+import { RouteNames } from '../../routes'
+import { smBreakpoint } from '../../styling/breakpoints'
+
+const ButtonsContainer = styled.div`
+  & {
+    display: flex;
+    align-items: center;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    
+    @media (min-width: ${smBreakpoint}) {
+      align-items: flex-start;
+    }
+  }
+  
+  > div {
+    &:first-of-type, &:nth-of-type(3) {
+      flex: 1;
+      
+      a {
+        width: 100%;
+      }
+    }
+    
+    &:nth-of-type(2) {
+      flex: 3;
+    }
+  }
+`
+
+
+const BlogListNavigationLink = ({ buttonText, currentPage, page }) => (
+  <Link
+    route={RouteNames.Blog}
+    passHref
+    params={{
+      page,
+    }}
+  >
+    <Button isActive={currentPage && (currentPage === page)}>{buttonText || page}</Button>
+  </Link>
+)
+
+BlogListNavigationLink.propTypes = {
+  page: PropTypes.number.isRequired,
+  currentPage: PropTypes.number,
+  buttonText: PropTypes.string,
+}
+
+BlogListNavigationLink.defaultProps = {
+  buttonText: undefined,
+  currentPage: undefined,
+}
+
+
+const PageButtonsContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  
+  > * {
+    display: none;
+    margin-left: 0.5rem;
+    margin-right: 0.5rem;
+    
+    @media (min-width: ${smBreakpoint}) {
+      display: block;
+    }
+  }
+`
+
+const BlogListNavigationButtons = ({
+  previousPageButtonText,
+  nextPageButtonText,
+  page,
+  isLastPage,
+  totalNumberOfPages,
+}) => {
+  const alterButtonIfLastPage = buttonPage => (isLastPage ? buttonPage - 1 : buttonPage)
+  const isFirstPage = page === 1
+  const isNotFirstPage = !isFirstPage
+  const isNotLastPage = !isLastPage
+  const moreThanOnePage = totalNumberOfPages !== 1
+  const threeOrMorePages = totalNumberOfPages >= 3
+
+  const firstPageButton = alterButtonIfLastPage(isNotFirstPage ? page - 1 : page)
+  const secondPageButton = alterButtonIfLastPage(isNotFirstPage ? page : page + 1)
+  const thirdPageButton = alterButtonIfLastPage(isNotFirstPage ? page + 1 : page + 2)
+
+  return (
+    <ButtonsContainer>
+      <div>
+        {
+          isNotFirstPage && (
+            <BlogListNavigationLink page={page - 1} buttonText={previousPageButtonText} />
+          )
+        }
+      </div>
+      <PageButtonsContainer>
+        {moreThanOnePage && (
+          <BlogListNavigationLink currentPage={page} page={firstPageButton} />
+        )}
+        <BlogListNavigationLink currentPage={page} page={secondPageButton} />
+        {threeOrMorePages && (
+          <BlogListNavigationLink currentPage={page} page={thirdPageButton} />
+        )}
+      </PageButtonsContainer>
+      <div>
+        {isNotLastPage && (
+          <BlogListNavigationLink page={page - 1} buttonText={nextPageButtonText} />
```

Comment:
We could annotate the Next and Previous links with `rel="next"` and `rel="prev"` for SEO purposes. See https://www.contentkingapp.com/academy/pagination/